modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceAccelerators.testcase

```diff
@@ -14,93 +14,334 @@ definition {
 		CommerceConfiguration.commerceTearDown(doLogout = "true");
 	}
 
-	@description = "This is a functional test for COMMERCE-8145 - Without an account, add to cart and quantity selector are disabled for admin user"
+	@description = "This is a test for COMMERCE-8145 and COMMERCE-10783. The add to cart button and quantity selector are disabled for the default admin user on the default B2B scenario"
```

Comment:
It could be better to specify the 'no account' factor:
"This is a test for COMMERCE-8145 and COMMERCE-10783. The add to cart button and quantity selector are disabled for the default admin user in a default B2B scenario when no account is created."